### PR TITLE
Feat: Home page

### PR DESCRIPTION
**Pull request by JuanLugoS**
Description:

-     Add Home Screen

Demo:

https://github.com/ISIS3510-202320-Team13/Flutter/assets/60227071/b0582755-9fc8-44a5-bfea-4daae8b7e906

### CODE REVIEW

File: assets/map_style.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "featureType": "poi",
+    "stylers": [
+      {
+        "visibility": "off"
+      }
+    ]
+  }
+]
```
Comment (Crola1702) [inline assets/map_style.json:10]: Please organize these files in a `map/` folder

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "*.yaml": "home-assistant"
+    }
+}
```
Comment (Crola1702) [inline .vscode/settings.json]: Please don't add `.vscode` settings to the project. Add them to your `.gitignore` or project's `.gitignore`

File: assets/map_style.txt [hunk 3]
```diff
@@ -0,0 +1,209 @@
+[
```
Comment (Crola1702) [inline assets/map_style.txt:1]: Same as above (Please organize these files in a `map/` folder)

File: lib/ui/home/home_page.dart [hunk 4]
```diff
@@ -0,0 +1,246 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class HomePage extends StatelessWidget {
+  late GoogleMapController mapController;
+  final LatLng _center = const LatLng(4.602796, -74.065841);
```
Comment (Crola1702) [inline lib/ui/home/home_page.dart:8]: What are those magic numbers in the `_center` attribute?

File: lib/ui/home/home_page.dart [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Color colorB1 = Color(int.parse("0961AD", radix: 16) + 0xFF000000);
+    Color colorB2 = Color(int.parse("2597FA", radix: 16) + 0xFF000000);
+    Color colorB3 = Color(int.parse("40A6FF", radix: 16) + 0xFF000000);
+
+    Color colorY1 = Color(int.parse("B89865", radix: 16) + 0xFF000000);
```
Comment (Crola1702) [inline lib/ui/home/home_page.dart]: Please add these colors to [theme/theme_constants](https://github.com/ISIS3510-202320-Team13/Flutter/blob/main/lib/ui/theme/theme_constants.dart)

File: lib/ui/home/home_page.dart [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+          height: 200.0,
+          child: Column(
+            children: [
+              Text(
+                'Buscar en:',
+                style: TextStyle(
+                  fontSize: 15,
+                  color: colorB1,
+                  decoration: TextDecoration.none,
+                ),
```
Comment (Crola1702) [inline lib/ui/home/home_page.dart:126]: Consider adding this TextStyle to [theme/theme_constants](https://github.com/ISIS3510-202320-Team13/Flutter/blob/main/lib/ui/theme/theme_constants.dart)

**Review by Crola1702 — CHANGES_REQUESTED** [review]
Looks Good. PTAL some comments of file organization.

File: assets/map_style.json (hunk 1 above)
Comment (JuanLugoS) [reply assets/map_style.json:10]: At the end is an asset, makes no sense do a complete folder 2 files

File: assets/map_style.txt (hunk 3 above)
Comment (JuanLugoS) [reply assets/map_style.txt:1]: At the end is an asset, makes no sense do a complete folder 2 files

File: lib/ui/home/home_page.dart (hunk 5 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart]: Done

File: assets/map_style.json (hunk 1 above)
Comment (Crola1702) [reply assets/map_style.json:10]: Keep in mind that the project will grow, and the assets folder might be filled with a lot of files.

File: lib/ui/home/home_page.dart (hunk 4 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart:8]: The app isnt ready to ask for location. Is a initial latitude and logitud

File: .vscode/settings.json (hunk 2 above)
Comment (JuanLugoS) [reply .vscode/settings.json]: Done

File: lib/ui/home/home_page.dart (hunk 6 above)
Comment (JuanLugoS) [reply lib/ui/home/home_page.dart:126]: Won´t use this style anymore

**Review by Crola1702 — APPROVED** [review]
LGTM. Please use "Squash and Merge" to shrink the commit history

**Pull request merged**